src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -890,7 +890,7 @@ static int xmc_get_data(struct platform_device *pdev, enum xcl_group_kind kind,
 	struct xcl_board_info *bdinfo = NULL;
 	struct xocl_xmc *xmc = platform_get_drvdata(pdev);
 
-	if (XMC_PRIVILEGED(xmc) && !xmc->mgmt_binary)
+	if (XMC_PRIVILEGED(xmc) && !xmc->mgmt_binary && !autonomous_xmc(pdev))
```

Comment:
This seems changing existing logic of XOCL_DSAFLAG_SMARTN?

Reply:
I've confirmed this with @chienwei-lan. SMARTN is OK
1) SMARTN does not care about sensor data